.evergreen/config.yml

```diff
@@ -101,6 +101,25 @@ functions:
              cp ./install/libmongocrypt/bin/mongocrypt.dll c:/libmongocrypt/bin
              cp ./install/libmongocrypt/include/mongocrypt/*.h c:/libmongocrypt/include
              export PATH=$PATH:/cygdrive/c/libmongocrypt/bin
+          elif [ "Darwin" = "$(uname -s)" ]; then
+             # TODO (GODRIVER-2442): Once libmongocrypt compiles on macOS 10.15, remove this elif block.
+             echo "fetching build for Darwin ... begin"
+             mkdir -p install/libmongocrypt
+             mkdir libmongocrypt-all
+             cd libmongocrypt-all
+             # LIBMONGOCRYPT_COMMIT is for 1.5.0-rc0.
+             LIBMONGOCRYPT_COMMIT="75dff76f3268c43ad7213bb8303688461aaabb96"
+             # The following URL is published from the upload-all task in the libmongocrypt Evergreen project.
+             curl https://mciuploads.s3.amazonaws.com/libmongocrypt/all/master/$LIBMONGOCRYPT_COMMIT/libmongocrypt-all.tar.gz -o libmongocrypt-all.tar.gz
+             tar -xf libmongocrypt-all.tar.gz
+             cd ..
+             mv libmongocrypt-all/macos/include ./install/libmongocrypt
+             mv libmongocrypt-all/macos/lib ./install/libmongocrypt
+             rm -rf libmongocrypt-all
+             rm libmongocrypt-all.tar.gz
+             # Fix prefix in pkg-config prefix path.
+             sed -i "" -E "s+prefix=.*+prefix=$(pwd)/install/libmongocrypt+" ./install/libmongocrypt/lib/pkgconfig/libmongocrypt.pc
```

Comment:
Love `sed` haha. What exactly is this "fixing" the prefix path to?

Reply:
The package config prefix includes the install path from the host that builds.

This is the contents of this tarball's macos/lib/pkgconfig/libmongocrypt.pc:

```
Name: mongocrypt
Description: The libmongocrypt client-side field level encryption library.
Version: 1.5.0-rc0
Requires: 
Requires.private: 
prefix=/System/Volumes/Data/data/mci/253c2ab5479da6c90e38aaa963434d83/install/libmongocrypt
includedir=${prefix}/include/mongocrypt
libdir=${prefix}/lib
Libs: -L${libdir} -lmongocrypt
Cflags: -I${includedir}
```

The path is included in the output of pkg-config:
```
% export PKG_CONFIG_PATH=./macos/lib/pkgconfig/ 
% pkg-config --cflags --libs libmongocrypt
-I/System/Volumes/Data/data/mci/253c2ab5479da6c90e38aaa963434d83/install/libmongocrypt/include/mongocrypt -L/System/Volumes/Data/data/mci/253c2ab5479da6c90e38aaa963434d83/install/libmongocrypt/lib -lmongocrypt
```

Reply:
Cool thanks for the explanation.